ui/app/selectors/custom-gas.js

```diff
@@ -264,29 +241,13 @@ export function getRenderableGasButtonData(
           conversionRate,
         )
       : '',
-    timeEstimate: fastWait && getRenderableTimeEstimate(fastWait),
```

Comment:
The `getRenderableTimeEstimate` function is now no longer used, nor is `getTimeEstimateInSeconds` and `formatTimeEstimate`